integration-testing/src/test/java/org/triplea/lobby/server/db/dao/BadWordsDaoTest.java

```diff
@@ -48,17 +49,12 @@ void removeBadWord() {
   @ParameterizedTest
   @MethodSource
   void badWordContains(final String badWord) {
-    assertThat(badWordsDao.containsBadWord(badWord), is(1));
-  }
-
-  @SuppressWarnings("unused")
-  private static List<String> notBadWordContains() {
-    return List.of("zz", "", "some word not containing any bad words");
+    assertThat(badWordsDao.containsBadWord(badWord), is(true));
   }
 
   @ParameterizedTest
-  @MethodSource
+  @ValueSource(strings = {"zz", "", "some word not containing any bad words"})
```

Comment:
```suggestion
  @ValueSource(strings = {"zz", "", "some string not containing any bad words"})
```
"word not containing word" sounds somewhat wrong to me 🤔